### PR TITLE
Employee contract new

### PR DESCRIPTION
**Pull request by ParkwayBE**
updated version including all contracts overview

### CODE REVIEW

**Comment by ThomasFrans** [comment]
You've already merged this I think.

**Comment by ParkwayBE** [comment]
i created this branch with the same name then the last one. this needs to be merged since there are new changes.

**Review by ThomasFrans — APPROVED** [review]

**Comment by ThomasFrans** [comment]
I think I marked this as approved but forgot to actually merge it. I'm very sorry if that's the case :/ It's hard to keep up with all the PRs sometimes.

**Comment by ParkwayBE** [comment]
Updated version I consider this feature done.

**Review by ThomasFrans — APPROVED** [review]

**Comment by ThomasFrans** [comment]
Still needs to be formatted before it can be merged.

**Comment by ParkwayBE** [comment]
yes im working on 1 last thing that came to mind. will format after.

**Comment by ParkwayBE** [comment]
I should have another column in 'employee_contracts' table to save the salary per contract. I created a discord thread in the right channel. For now the contract gets the salary from the 'employees' table. But that one is static. So we should update that one so it changes its value when the contract expires and a new one goes in with a different salary. I noticed other people like Mantriq are using this value of the salary column in 'employees_contract' so we cant just removed it and put it somewhere else. @ThomasFrans

**Comment by ParkwayBE** [comment]
this should be ready to merge now?

**Comment by ThomasFrans** [comment]
> this should be ready to merge now?

There are merge conflicts.

**Comment by ParkwayBE** [comment]
> > this should be ready to merge now?
> 
> There are merge conflicts.
should I resolve these? or?

**Comment by ThomasFrans** [comment]
Yes

**Comment by ParkwayBE** [comment]
There are things that are broken now.. trying to fix this

**Comment by ParkwayBE** [comment]
> > this should be ready to merge now?
> 
> There are merge conflicts.

I fixed them

File: routes/web.php [hunk 1]
```diff
@@ -66,8 +65,13 @@
...
-        Route::post('/employees/{employee}', 'update')->name('employees.update');
-        Route::post('/employee_add_contract_done', 'contract_save')->name('employee-add-contract');
+        Route::post('/employees/{employee}', 'update')->name('employees.update')->middleware('permission:add_employee');
+        Route::get('/employee_add_contract', 'contract_index')->name('contract.index')->middleware('permission:add_employee');
+        Route::post('/employee_add_contract_done', 'contract_save')->name('employee-add-contract')->middleware('permission:add_employee');
+        Route::get('/employee_view_contracts', 'view_contracts_index')->name('employee-view-contracts')->middleware('permission:add_employee');
+        Route::post('/employee_view_contracts/details', 'employeeContractDetails')->name('employee-contract-details')->middleware('permission:add_employee');
+        Route::get('/employee_contract_search', 'searchEmployeeContract')->name('employee-contract-search')->middleware('permission:add_employee');
+        Route::post('/employee_view_contracts/pdf', 'createEmployeeContractPDF')->name('employee-download-contract')->middleware('permission:add_employee');
```
Comment (ThomasFrans) [inline routes/web.php:74]: The permissions aren't correct.

**Review by ThomasFrans — CHANGES_REQUESTED** [review]

**Comment by ParkwayBE** [comment]
changed those permission and added a new permission to app service provider for changing contracts

**Review by ThomasFrans — APPROVED** [review]

**Pull request merged**